sw/airborne/modules/radio_control/rc_datalink.h

```diff
@@ -29,48 +29,76 @@
 
 #include "std.h"
 
-#define RC_DL_NB_CHANNEL 5
+#define RC_DL_NB_CHANNEL 11
 
 /**
  * Redefining RADIO_*
  * Do not use with radio.h (ppm rc)
  */
-#define RADIO_ROLL      0
-#define RADIO_PITCH     1
-#define RADIO_YAW       2
-#define RADIO_THROTTLE  3
-#define RADIO_MODE      4
-#define RADIO_AUX1      5
-#define RADIO_AUX2      6
-#define RADIO_AUX4      7
-#define RADIO_AUX6      8
-#define RADIO_AUX5      9
+#define RADIO_ROLL      0  // Default: Roll 
+#define RADIO_PITCH     1  // Default: Pitch
+#define RADIO_YAW       2  // Default: Yaw
+#define RADIO_THROTTLE  3  // Default: Thrust
+#define RADIO_MODE      4  // Default: MODE
+#define RADIO_AUX1      5  // Default: KILL
+#define RADIO_AUX2      7  // Default: NA
+#define RADIO_AUX4      8  // Default: NA
+#define RADIO_AUX5      9  // Default: NA
+#define RADIO_AUX6      10 // Default: NA
+#define RADIO_AUX7      6  // Default: AP_MODE
```

Comment:
I don't yet this ordering ? Why AUX7 is also AP_MODE if there is already a MODE channel ?

Reply:
I have added some extra description to the definitions.

Reply:
Making RADIO_AUX7 channel 6 means that in the joystick file you do not have to add any extra zeros to the array. This also results in a smaller array.

Reply:
see : https://github.com/paparazzi/paparazzi/blob/master/conf/autopilot/rotorcraft_oneloop_switch.xml

Reply:
If the naming is confusing it can be changed!